src/NuGet.Core/NuGet.ProjectModel/JsonPackageSpecReader.cs

```diff
@@ -181,6 +183,34 @@ private static NuGetVersion SpecifySnapshot(string version, string snapshotValue
             return new NuGetVersion(version);
         }
 
+        private static PackOptions GetPackOptions(PackageSpec packageSpec, JObject rawPackageSpec)
+        {
+            var rawPackOptions = rawPackageSpec.Value<JToken>("packOptions") as JObject;
+            if (rawPackOptions == null)
+            {
+                return new PackOptions();
+            }
+
+            var rawPackageType = rawPackOptions["packageType"];
```

Comment:
We are not going to support multiple package types yet. However I agree that we should parse the multiple types and, at a higher level, reject them if there are multiple. This would align with what is happening for the .nuspec parsing. Multiple values are parsed out but `NuGetPackageManager` code rejects multiple (or unexpected values, in this case) upon installation.